lib/bn.js

```diff
@@ -459,6 +460,11 @@ BN.prototype.toJSON = function toJSON() {
   return this.toString(16);
 };
 
+BN.prototype.toBuffer = function toBuffer(endian, length) {
```

Comment:
I think implementing ```toArrayLike``` should be left as an exercise for a separate pull request, because it is about optimisation.